src/parse-text.js

```diff
@@ -0,0 +1,117 @@
+var moment = require('moment');
```

Comment:
Is it possible to update some of this parsing logic to allow `moment` to do most of the hard work for us? I'm wondering if we could use a regular expression to parse out the two dates then pass those values to `moment` and format them the way we want. The regex could look at `(?:.*[AM|PM] )` or something like that.